config/locales/mailers/en.yml

```diff
@@ -120,6 +120,36 @@ en:
       subject: "[%{app_name}] Your work has been deleted by an admin"
       text:
         tos_violation: If it's possible your work violated the Archive's Terms of Service, please contact our Policy & Abuse team (%{contact_abuse_url}).
+    admin_hidden_bookmark_notification:
+      access: While your bookmark is hidden, you will still be able to access it through the link provided above, but it will not be listed on your bookmarks page, and it won't be available to other users of the Archive.
+      check_email: Please check your email, including your spam folder, as the Policy & Abuse team may have already contacted you explaining why your bookmark was hidden.
+      contact_abuse: contact Policy & Abuse
+      help:
+        html: If you are uncertain why your bookmark was hidden, and you have not received further communication regarding this matter, please %{contact_abuse_link} directly.
+        text: "If you are uncertain why your bookmark was hidden, and you have not received further communication regarding this matter, please contact Policy & Abuse directly: %{contact_abuse_url}."
+      hidden:
+        html: Your bookmark of %{title} has been hidden by the Policy & Abuse team and is no longer publicly accessible.
+        text: Your bookmark of "%{title}" (%{bookmark_link}) has been hidden by the Policy & Abuse team and is no longer publicly accessible.
```

Comment:
In the text version of the email this is a plain URL so the variable name should `bookmark_url`, not _link

(Needs the change in admin_hidden_bookmark_notification.text.erb too)